vmd-api/api/openapi.yaml

```diff
@@ -0,0 +1,336 @@
+openapi: 3.1.0
+info:
+  contact:
+    email: nikita.bazulin@unikie.com
+  description: This is Control Panel API based on the OpenAPI 3.1 specification of
+    the GHAF project
+  license:
+    name: Apache License 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  title: Virtual Machine Manager API of the GHAF project - OpenAPI 3.1
+  version: 1.0.1
+externalDocs:
+  description: Find out more about the GHAF project
+  url: https://github.com/tiiuae/ghaf
+servers:
+- description: The primary server
+  url: https://localhost/api/v1
+- description: The test server
+  url: https://localhost/api/test/v1
+tags:
+- description: Operations of Virtual Machines
+  name: vm
+paths:
+  /vm/list:
+    get:
+      description: Return a IDs list of all Virtual Machines
```

Comment:
s/Return a IDs list of all Virtual Machines/Gets list of IDs for all virtual machines/

Reply:
Why use Gets, but not Returns?

I was following the style in the reference example:
https://github.com/swagger-api/swagger-petstore/blob/master/src/main/resources/openapi.yaml#L325

Reply:
My 2 cents:
- shorter (preferred)
- aligned with the verb (get)

Reply:
Done